test.sh

```diff
@@ -136,13 +127,15 @@ while getopts lueciosvgnhp:f:-: OPT; do
   case "$OPT" in
     l | lints )                      RUN+=("lints") ;;
     u | unit )                       RUN+=("unit") ;;
+    v | unit-verbose )               UNIT_FLAGS+=("-v") ;;
+    w | unit-without-cache )         UNIT_FLAGS+=("-count=1") ;;
     p | unit-test-package )          check_arg; UNIT_PACKAGES+=("${OPTARG}") ;;
-    e | enable-race-detection )      RACE="true" ;;
```

Comment:
We can't quite get rid of `$RACE` altogether as integration-test.py still depends on it: https://github.com/letsencrypt/boulder/blob/5b227b8615ec1cc880386be3bcc95fa818415fce/test/integration-test.py#L34